modules/raycast/SCsub

```diff
@@ -80,9 +80,16 @@ if env["builtin_embree"]:
     env_thirdparty.disable_warnings()
     env_thirdparty.add_source_files(thirdparty_obj, thirdparty_sources)
 
-    if env["arch"] != "x86_64" or env.msvc:
-        # Embree needs those, it will automatically use SSE2NEON in ARM
-        env_thirdparty.Append(CPPDEFINES=["__SSE2__", "__SSE__"])
+    # Set x86 CPU instruction sets to use when building Embree's own intrinsics.
+    # Keep this in sync with Godot's main SConstruct file.
+    # This is only needed on MSVC, as GCC/Clang will set those defines automatically
+    # according to compiler instruction set flags.
+    if env.msvc:
```

Comment:
Actually wait, this changes the logic. Previously it would define `_SSE_` and `_SSE2_` on all platforms but x86_64.

So on arm64, wasm32, etc.
We're changing this here.

Reply:
I've pushed a commit that reverts to the old logic while adding the new flags for x86_64 MSVC only.